Makefile

```diff
@@ -1,10 +1,25 @@
 PACKAGE_NAME = image-builder
 
+.PHONY: help
+help:
+	@echo "make [TARGETS...]"
+	@echo
+	@echo "This is the maintenance makefile of image-builder. The following"
+	@echo "targets are available:"
+	@echo
+	@echo "    help:               Print this usage information."
+	@echo "    build:              Build the project from source code"
+	@echo "    test:               Run tests"
+	@echo "    run:                Run the project on localhost"
+
 .PHONY: build
 build:
 	go build -o image-builder ./cmd/image-builder/
 	go build -o gen-oscap ./cmd/oscap
 	go build -o image-builder-migrate-db-tern ./cmd/image-builder-migrate-db-tern/
+
+.PHONY: test
+test:
```

Comment:
db-test in the github workflow tests.yml also needs to be updated to run make test, the test is failing because ./image-builder-db-test is now missing (it uses make build).